core/src/main/java/org/elasticsearch/action/admin/indices/validate/query/TransportValidateQueryAction.java

```diff
@@ -75,17 +76,21 @@
 
     private final BigArrays bigArrays;
 
+    private FetchPhase fetchPhase;
+
     @Inject
     public TransportValidateQueryAction(Settings settings, ThreadPool threadPool, ClusterService clusterService,
                                         TransportService transportService, IndicesService indicesService,
                                         ScriptService scriptService, PageCacheRecycler pageCacheRecycler,
-                                        BigArrays bigArrays, ActionFilters actionFilters, IndexNameExpressionResolver indexNameExpressionResolver) {
+ BigArrays bigArrays, ActionFilters actionFilters,
+            IndexNameExpressionResolver indexNameExpressionResolver, FetchPhase fetchPhase) {
```

Comment:
I just pushed a commit to address these issues. Hopefully I managed to catch them all